content/library/advanced-features/connection-to-data.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Connecting to data
+slug: /library/advanced-features/connecting-to-data
+---
+
+# Connecting to data
+
+Most Streamlit apps need some kind of data or API access to be useful - either retrieving data to view or saving the results of some user action. This data or API is often part of some remote service, database, or other data source.
+
+Anything you can do with Python, including data connections, will generally work in Streamlit. Streamlit's [tutorials](/knowledge-base/tutorials/databases) are a great starting place for many data sources. However:
```

Comment:
Can we bold the first sentence like it is in Notion?

Reply:
Fixed on 4164294, but I vote heavily against bolding. It looks terrible 😂 And since it's bang on in the first section, I'm certain even users who skim the docs won't miss the sentence => we should not bold the sentence.